caravel/assets/javascripts/explore.js

```diff
@@ -334,9 +338,14 @@ function initExploreView() {
     add_filter(undefined, "having");
   });
 
-  $(".query").click(function () {
-    query(true);
-  });
+  const queryAndSaveBtnsEl = document.getElementById('js-query-and-save-btns');
+  ReactDOM.render(
+    <QueryAndSaveBtns
```

Comment:
this the line that failed, I think webpack needs a `.jsx` extension to get the right loader.